docs/manual/manual_timeout.rst

```diff
@@ -0,0 +1,158 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _manual_timeout:
+
+***********************************************
+Timeout
+***********************************************
+Vertiq modules support a timeout that will trigger if they do not receive a new command within a configurable period of time when they are spinning. 
+This is intended primarily as a safety feature, to safely stop the module if communication is lost while the module is spinning. Modules will also always 
+disarm when a timeout occurs, to leave them in a safe state following a timeout.
+
+.. _timeout_module_support:
+
+Module Support
+===============
+
+Speed Modules
+**************
+
+.. table:: Speed Module Support for Configurable Timeout
+
+	+-------------+------------------------------------+
+	| Module      | Configurable Timeout Support       |
+	+-------------+------------------------------------+
+	| Vertiq 8108 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 4006 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+	| Vertiq 2306 | .. centered:: |:white_check_mark:| |
+	+-------------+------------------------------------+
+
+Servo Modules
+**************
+Servo modules do not support the full range of configurable timeout features described in this section. They do support a timeout, and the period of the timeout is configurable.
+The timeout behavior, timeout song playback options, and timeout meaning described in this section are not supported on servo modules. They will always 
+continuously play the timeout song when they timeout.

```

Comment:
uhhhh...is this actually true? Currently each of the servo modules has a "TODO:: Add timeout stuff when implemented on multiturn" in their release mains. Does any of them actually play a song?

Reply:
Yeah I guess not. I just thought they did